charts/sophora-schema-docs/templates/deployment.yaml

```diff
@@ -22,47 +22,54 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       containers:
-        - name: schemadoc
+        - name: schema-docs
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
-          args:
-            {{if eq .Values.schemadocs.mode "static"}}
-            [ "--static", "--out", "/out/schemadocs" ]
-            {{else}}
-            [ "--dynamic" ]
-            {{end}}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args: ["--dynamic"]
           volumeMounts:
-          {{if eq (.Values.image.tag | default .Chart.AppVersion) "4.0.0" }}
-            - mountPath: /data
-          {{else}}
-            - mountPath: /config
-          {{end}}
-              name: config
-            - mountPath: /out
-              name: output-folder
+            - name: config
+              # 4.0.0 is the only version that uses WORKDIR /data
+              {{if eq (.Values.image.tag | default .Chart.AppVersion) "4.0.0" }}
```

Comment:
By adding a `-` after `{{` you remove unnecessary whitespace after rendering the yaml file. Not super important but good practice.

Reply:
I've made modifications to remove said whitespace.